src/ZoneCommon/Pool/XAssetInfo.h

```diff
@@ -61,6 +53,16 @@ class XAssetInfoGeneric
     XAssetInfoGeneric(XAssetInfoGeneric&& other) noexcept = default;
     XAssetInfoGeneric& operator=(const XAssetInfoGeneric& other) = default;
     XAssetInfoGeneric& operator=(XAssetInfoGeneric&& other) noexcept = default;
+
+    static std::string NormalizeAssetName(std::string input);
```

Comment:
Localize assets seem to be dumped lowercase now which is not intended

Reply:
I ended up not normalizing the name of the asset in the XAssetInfo, so this is solved now